Library/ViewModels/RewardsCollectionViewModel.swift

```diff
@@ -96,21 +94,24 @@ public final class RewardsCollectionViewModel: RewardsCollectionViewModelType,
       selectedRewardFromId,
       refTag
     )
-    .filter { project, _, _ in project.state == .live }
-    .map { project, reward, refTag in
-      PledgeData(project: project, reward: reward, refTag: refTag)
+    .filter { project, reward, _ in
+      project.state == .live && !userIsBacking(reward: reward, inProject: project)
+    }
+    .map { project, reward, refTag -> (PledgeViewData, Bool) in
+      let data = PledgeViewData(
+        project: project,
+        rewards: [reward],
+        selectedQuantities: [reward.id: 1],
```

Comment:
Is this going to change in the future? Should we be setting this to 1?

Reply:
Well-spotted. This is just a somewhat contrived way of abstracting the behaviour for regular rewards and add-ons based rewards. We know (for now) that the base reward will always be selected once.